docs/docs/integrations/providers/comet_tracking.ipynb

```diff
@@ -192,49 +160,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from langchain.agents import initialize_agent, load_tools\n",
-    "from langchain_community.callbacks import CometCallbackHandler\n",
-    "from langchain_core.callbacks import StdOutCallbackHandler\n",
-    "from langchain_openai import OpenAI\n",
-    "\n",
-    "comet_callback = CometCallbackHandler(\n",
-    "    project_name=\"comet-example-langchain\",\n",
-    "    complexity_metrics=True,\n",
-    "    stream_logs=True,\n",
-    "    tags=[\"agent\"],\n",
-    ")\n",
-    "callbacks = [StdOutCallbackHandler(), comet_callback]\n",
-    "llm = OpenAI(temperature=0.9, callbacks=callbacks)\n",
+    "from opik.integrations.langchain import OpikTracer\n",
     "\n",
-    "tools = load_tools([\"serpapi\", \"llm-math\"], llm=llm, callbacks=callbacks)\n",
-    "agent = initialize_agent(\n",
-    "    tools,\n",
-    "    llm,\n",
-    "    agent=\"zero-shot-react-description\",\n",
-    "    callbacks=callbacks,\n",
-    "    verbose=True,\n",
-    ")\n",
-    "agent.run(\n",
-    "    \"Who is Leo DiCaprio's girlfriend? What is her current age raised to the 0.43 power?\"\n",
-    ")\n",
-    "comet_callback.flush_tracker(agent, finish=True)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Scenario 4: Using Custom Evaluation Metrics"
+    "opik_tracer = OpikTracer(\n",
+    "    tags=[\"langchain\"],\n",
+    "    metadata={\"use-case\": \"documentation-example\"}\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `CometCallbackManager` also allows you to define and use Custom Evaluation Metrics to assess generated outputs from your model. Let's take a look at how this works. \n",
+    "### Access Logged Traces\n",
     "\n",
-    "\n",
-    "In the snippet below, we will use the [ROUGE](https://huggingface.co/spaces/evaluate-metric/rouge) metric to evaluate the quality of a generated summary of an input prompt. "
+    "You can retreieve Traces logged to Opik with the `created_traces`  method. This can be helpful if you need to log feedback scores or update a trace. "
```

Comment:
```suggestion
    "You can retrieve Traces logged to Opik with the `created_traces`  method. This can be helpful if you need to log feedback scores or update a trace. "
```